packages/ens-utils/tsup.config.ts

```diff
@@ -2,9 +2,10 @@ import { defineConfig } from "tsup";
 
 export default defineConfig({
```

Comment:
Would it make sense to create another internal package that effectively provides a common config that all packages and apps can then make use of?